python-lib/dku_utils/config_parser.py

```diff
@@ -0,0 +1,18 @@
+# Provide some utility methods to parse the saved configuration, clean it, normalize it and return in a predifined format (ex: command line args)
```

Comment:
Typo: predefined

Reply:
We can split the line after `predefined` (too avoid the 'line is too long' warning.